chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+# 31 Usage Profile section
```

Comment:
Note that SPDX 2.3 doesn't yet have a "profile" concept, and if I'm understanding this correctly, these are all fields that would be included on a Package rather than being in their own section. If that's correct, then I assume this would all go into the [Package Information section](https://github.com/spdx/spdx-spec/blob/development/v2.3/chapters/package-information.md) as optional fields rather than in a separate standalone chapter?